infrastructure/linkerd/README.md

```diff
@@ -29,6 +29,6 @@ Generate a Kubernetes secret with the Linkerd certs:
 ```sh
 kubectl -n linkerd create secret generic linkerd-certs \
 --from-file=ca.crt --from-file=issuer.crt \
---from-file=issuer.key -oyaml --dry-run=client \
+--from-file=issuer.key -oyaml --dry-run=true \
```

Comment:
Was this flag-value change intentional?
I think you may need to update your version of `kubectl`:
```
W0526 10:08:12.756453   22975 helpers.go:571] --dry-run=true is deprecated (boolean value) and can be replaced with --dry-run=client.
```

Reply:
Doh!
It's my kubebuilder path that gives me a old kubectl version, I will have to update my PATH order.
Thanks for the info